ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/templates/Template5.kt

```diff
@@ -242,6 +242,8 @@ private fun Title(
         textAlign = TextAlign.Left,
         text = state.selectedLocalization.title,
         color = state.templateConfiguration.getCurrentColors().text1,
+        modifier = Modifier
+            .fillMaxWidth(),
```

Comment:
I believe it should but just to check, this look good on landscape?

Reply:
Yeah 👍🏻